src/components/Header.tsx

```diff
@@ -0,0 +1,82 @@
+import { scrollToID } from '@/utils/scroll';
+import React from 'react';
+import { Link as FeatherLinkIcon } from "react-feather";
+import styled from 'styled-components';
+
+const StyledLinkIcon = styled.a`
+  text-decoration: none;
+  position: absolute;
+  width: 3rem;
+  height: 2rem;
+  display: none;
+  align-items: center;
+  left: -2rem;
+  &:hover {
+    text-decoration: underline;
+  }
+`;
+
+const StyledLinkHeading = styled.a`
+  text-decoration: none;
+  position: absolute;
+  font-weight: bold;
+  &:hover {
+    text-decoration: underline;
+  }
+`;
+
+const BaseStyledHeading = styled.div`
+  display: flex;
+  align-items: center;
+  position: relative;
+  padding: 1.5rem 0 0;
+  margin-bottom: 2rem;
+
+  &:hover {
+    ${StyledLinkIcon} {
+      display: flex;
+    }
+    @media (max-width: 1300px) {
+      ${StyledLinkIcon} {
+        display: none;
+      }
+    }
+  }
+`;
```

Comment:
Curious here, is it not possible to use something like Tailwind CSS?

Reply:
Definitely, but I didn't wanna touch this as it works and I don't know tailwind, I can come back to stuff like this in the future when I'm more familiar.